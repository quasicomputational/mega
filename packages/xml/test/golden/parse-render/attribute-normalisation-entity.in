<!DOCTYPE foo SYSTEM "example:entity-set">
<foo bar="ok &NewLine; gotcha"></foo>
