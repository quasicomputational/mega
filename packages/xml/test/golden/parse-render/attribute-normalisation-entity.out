<foo bar="ok &#xA; gotcha"></foo>